charts/latest/azuredisk-csi-driver/values.yaml

```diff
@@ -72,6 +72,7 @@ controller:
   vmType: ""
   provisionerWorkerThreads: 100
   attacherWorkerThreads: 1000
+  detachOperationMinTimeoutInSeconds: 120
```

Comment:
should be 240 here? and actually it's not necessary to expose every value and setting in helm chart config, you could revert all changes in helm chart config in this PR, that would be easier for cherrypick.

Reply:
yeah, I can revert the changes for this PR.